src/elements/SingleReview.jsx

```diff
@@ -42,41 +53,79 @@ export const SingleReview = ({
       });
   };
 
+  if (!reviewLoaded) return "Loading Review ...";
+  if (!review) return <span className="error">404: couldn't find review</span>;
```

Comment:
This error line would be nicer if controlled by a error state but totally up to you